docs/examples/multiple-agent-shared-memory.md

```diff
@@ -56,7 +56,7 @@ mongo_driver = AzureMongoDbVectorStoreDriver(
 
 loader = Agent(
     tools=[
-        WebScraper()
+        WebScraper(off_prompt=True),
```

Comment:
Should be `False`.

Reply:
That's the nifty part about this example, it's a separate Agent using the same shared Task/Meta Memory instances. So this is correct.